### PR TITLE
[scripts] Update remove_scsi preprocess

### PR DESCRIPTION
**Pull request by SHtress**
* [x] Read PR [documentation](https://github.com/ostis-ai/sc-machine/blob/main/CONTRIBUTING.md)
* [x] Update changelog
* [x] Update documentation

### CODE REVIEW

**Review by KovalM — CHANGES_REQUESTED** [review]
Don't remove 'remove_scsi.py` script. We should folder with migrations scripts. Near builder, let's discuss about it

File: scripts/kb_scripts/remove_scsi.py [hunk 1]
```diff
@@ -3,32 +3,32 @@
...
 
 
-def remove_file_mentions(scsi_file, relative_path, buffer_file_list, tabulation_string) -> list:
-    for scsi_file_line in scsi_file:
-        if scsi_file_line.find(r'"file://') >= 0 and len(relative_path) > 0:
-            scsi_file_line = scsi_file_line[:scsi_file_line.find('//')+2] + \
-                         relative_path + scsi_file_line[scsi_file_line.find('//')+2:]
-        buffer_file_list.append(tabulation_string + scsi_file_line)
-    return buffer_file_list
+def process_scsi(scsi_file_path , structure_name) -> None:
```
Comment (NikitaZotov) [inline scripts/kb_scripts/remove_scsi.py:6]: May be to make tests to check work of this script?

**Review by NikitaZotov — CHANGES_REQUESTED** [review]

File: scripts/kb_scripts/remove_scsi.py (hunk 1 above)
Comment (SHtress) [reply scripts/kb_scripts/remove_scsi.py:6]: No need for tests

File: docs/changelog.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Update scsi preprocess
```
Comment (KovalM) [inline docs/changelog.md:30]:
```suggestion
- Make scsi pre-processor migrations script
```

File: scripts/kb_scripts/remove_scsi.py (hunk 1 above)
Comment (KovalM) [inline scripts/kb_scripts/remove_scsi.py:6]: Add comment about this script as migration script

**Review by KovalM — CHANGES_REQUESTED** [review]

**Review by NikitaZotov — APPROVED** [review]

**Pull request merged**